tests/unit/test_gapjunctions.py

```diff
@@ -0,0 +1,211 @@
+                        RINGTEST_DIR / "gapjunctions" / "test_g_pas_passive.hdf5"
+                    ),
+                    "manual_MEComboInfo_file": str(
+                        RINGTEST_DIR / "gapjunctions" / "test_holding_per_gid.hdf5"
+                    )
+                }
+            }
+        }
+    ],
+    indirect=True,
+)
+def test_gap_junction_corrections(capsys, create_tmp_simulation_config_file):
+    """test for gap junction calibration, the steps tested are similar to the thalamus publication
+    """
+
+    nd = Neurodamus(create_tmp_simulation_config_file)
+
+    assert SimConfig.beta_features == {
+        "gapjunction_target_population": "RingC",
+        "deterministic_stoch": True,
+        "procedure_type": "validation_sim",
+        "gjc": 0.2,
+        "load_g_pas_file": str(RINGTEST_DIR / "gapjunctions" / "test_g_pas_passive.hdf5"),
+        "manual_MEComboInfo_file": str(RINGTEST_DIR / "gapjunctions" / "test_holding_per_gid.hdf5"),
+    }
+
+    # check log
+    captured = capsys.readouterr()
+    ref = re.compile(
+        r"[\s\S]*Load user modification.*(CellDistributor: RingC).*\n"
+        r".*Set deterministic = 1 for StochKv\n"
+        r".*Set GJc = 0.2 for 2 gap synapses\n"
+        r".*Update g_pas to fit 0.2 -.*for 1 cells\n"
+        r".*Load holding_ic from manual_MEComboInfoFile.*for 1 cells\n[\s\S]*"
+    )
+    assert ref.match(captured.out)
+
+    # check gap junction parameters after user correction
+    cell_manager = nd.circuits.get_node_manager("RingC")
+    tgt_cellref = cell_manager.get_cellref(2001)
+    gj_manager = nd.circuits.get_edge_manager("RingC", "RingC", GapJunctionManager)
+    connection = next(gj_manager.get_connections(post_gids=[2003], pre_gids=[2001]))
+    assert connection.synapses[0].g == 0.2
+    assert tgt_cellref.soma[0](0.5).pas.g == 0.033
+    assert len(gj_manager.holding_ic_per_gid) == 1
+    iclamp = gj_manager.holding_ic_per_gid[2001]
+    assert "IClamp" in iclamp.hname()
+    npt.assert_allclose(iclamp.dur, 9e9)
+    npt.assert_allclose(iclamp.amp, -0.00108486, rtol=1e-5)
+    assert gj_manager.seclamp_per_gid == {}
+
+    # Assert simulation went well
+    nd.run()
+
+
+@pytest.mark.parametrize(
+    "create_tmp_simulation_config_file",
+    [
+        {
+            "simconfig_fixture": "ringtest_baseconfig",
+            "extra_config": {
+                "network": str(RINGTEST_DIR / "circuit_config_gj.json"),
+                "node_set": "ABC",
+                "beta_features": {
+                    "gapjunction_target_population": "RingC",
+                    "remove_channels": "all",
+                    "procedure_type": "find_holding_current",
+                    "gjc": 0.2,
+                    "vc_amp": str(RINGTEST_DIR / "gapjunctions" / "test_holding_voltage.hdf5"),
+                }
+            }
+        }
+    ],
+    indirect=True,
+)
+def test_gap_junction_corrections_otherfeatures(capsys, create_tmp_simulation_config_file):
```

Comment:
what are these misterious other features? Either split the test, find a better name or add the tested features in detail in a docstring. This name is mysterious! uuuuh

Reply:
if they are the beta_Features maybe you can add a docstring that links to the docs about these

Reply:
`beta_features` are not official, they should be moved to proper sections in the sonata simulation config file.  It was used for thalamus gap junction simulation (PhD work). We didn't have time to move them into the official SONATA spec at the end of BBP. In OBI, we don't have plan yet.